src/player.ts

```diff
@@ -2006,6 +2007,23 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * In iframe embed add title element with the entry name
+   * @returns {void}
+   * @private
+   */
+  private _addTitleOnIframe(): void {
+    if (window.self !== window.top) {
```

Comment:
looking at answers from the web, it looks like both window.top and window.frameElement can throw an error in case of being embedded in a cross domain parent page

maybe using referrer would work better